tests/scripts/helpers/utils.py

```diff
@@ -1514,3 +1514,15 @@ def get_metric_profile_dir():
     metric_profile_dir = base_dir / 'manifests' / 'autotune' / 'performance-profiles'
 
     return metric_profile_dir
+
+def validate_local_monitoring_recommendation_data_present(recommendations_json):
+    if recommendations_json[0]['experiment_type'] == NAMESPACE_EXPERIMENT_TYPE:
+        assert recommendations_json[0]['kubernetes_objects'][0]['namespaces']['recommendations']['data']
```

Comment:
Please add a check for the recommendation message as well

Reply:
Added check for recommendations message and updated the error msg as well